docs/howtos/Client Generation/09versioning.mdx

```diff
@@ -258,11 +258,29 @@ serviceClientClient.stableFunctionality(stableModel);
 
 ### Override to a specific version
 
-You can override the signature to return the api surface area for a specific api version.
+You can override the signature to return the api surface area for a specific api version, this change is done in `tspconfig.yaml`.
```

Comment:
I think I would add this after the sentence below about the example, and make it more parallel with the example below, with a similar sentence like 'In this example, you cna see how this change is made in tspconfig.yaml'